app/src/main/java/openfoodfacts/github/scrachx/openfood/views/HistoryScanActivity.java

```diff
@@ -83,7 +87,8 @@ public boolean onOptionsItemSelected(MenuItem item) {
                         .title(R.string.title_clear_history_dialog)
                         .content(R.string.text_clear_history_dialog)
                         .onPositive((dialog, which) -> {
-                            mHistoryProductDao.deleteAll();;
+                            mHistoryProductDao.deleteAll();
+                            ;
```

Comment:
Can you remove the duplicate ';'

Reply:
Must have existed from before. I will surely remove it . Thanks.

Reply:
@itchix  I have removed the duplicate . Please Review and Approve .